gordon-plugin/src/main/kotlin/com/banno/gordon/GordonTestTask.kt

```diff
@@ -59,11 +59,13 @@ internal abstract class GordonTestTask : DefaultTask() {
     private val installTimeoutMillis = project.extensions.getByType<GordonExtension>().installTimeoutMillis
     private val testTimeoutMillis = project.extensions.getByType<GordonExtension>().testTimeoutMillis
 
+    @get:Input
     @Option(option = "tests", description = "Comma-separated packages, classes, or methods.")
```

Comment:
This intentionally wasn't a task input because it is only used by `testFilters`, which is a task input.
```suggestion
    @Option(option = "tests", description = "Comma-separated packages, classes, or methods.")
```